tests/test_handler_r2_score.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import numpy as np
+import torch
+
+from monai.handlers import R2Score
+
+
+class TestHandlerR2Score(unittest.TestCase):
```

Comment:
Base on the ci :https://github.com/Project-MONAI/MONAI/actions/runs/10926409391/job/30330218578?pr=8093#step:8:13844

You need skip the tests if no ignite installed.